src/pages/static/PrivacyPolicy.js

```diff
@@ -1,29 +1,9 @@
 import React from 'react';
+import { MarkdownReader } from './MarkdownReader';
+import privacyPolicy from './PrivacyPolicy.md';
```

Comment:
Question: Is this going to return the path? I'm not sure how it works, don't we need to update a webpack loader or something? Just asking out of ignorance 😅

Reply:
Yeah, it returns the path relative to the webpack entry point, but actually the _bundled_ path. so if we `console.log(privacyPolicy)` here, we get:

```
/static/media/PrivacyPolicy.6c67e12d.md
```

...which ...works?

I admit I don't fully understand it myself, and can't find it documented anywhere. I would much, much rather add a Markdown webpack loader and just handle the text, rather than `fetch`ing a file from a path we've magically generated. But as far as I can tell, this project started from Create React App, and we'd have to eject to configure webpack. Not worth it just for this, imo.

Reply:
Ok, I understand this a little better this morning.

Create React App is [configured](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js) with webpack loaders for JS files (.js, .tsx, .mjs, .json...), CSS & SASS files and HTML files, and all other files are handled by [`file-loader`](https://github.com/webpack-contrib/file-loader).

`file-loader` emits the public URI of the imported file. Its main use case is with images, where the URI can be fed to the `img.src` attribute:

```
import kittenImage from "./images/kitten.png";

const Image = () => <img src={kittenImage} />
```

...but since we don't have a convenient HTML element that will just load our Markdown for us, our only option is to `fetch` the raw text, store it in state and let `remark` render it for us.

Reply:
I agree, we don't want to eject just jet. Thanks for the detailed explanation.